Automattic/wp-calypso
# Loads Jetpack Scan credentials status into settings page

#### Changes proposed in this Pull Request

* Fixes a bug where the credentials showed as not set for sites with only Jetpack Scan.

#### Testing instructions



* Create a connected JP site with Scan _only_ purchased.
* Visit Cloud > Site Settings.
* Verify credentials are not set.
* Enter valid credentials and click "Save Credentials".
* Verify that credentials are saved and UI is in state as below:

<img width="752" alt="Screen Shot 2020-10-15 at 12 10 27 PM" src="https://user-images.githubusercontent.com/1760168/96156970-b7dad580-0edf-11eb-8a6f-724bfdc78571.png">

Fixes 1164141197617539-as-1198190187182841.
